backend/app/usecase/shortlink/updater.go

```diff
@@ -33,39 +42,46 @@ type UpdaterPersist struct {
 // UpdateShortLink mutates a short link in the repository.
 func (u UpdaterPersist) UpdateShortLink(
 	oldAlias string,
-	update entity.ShortLink,
+	shortLinkInput entity.ShortLinkInput,
 	user entity.User,
 ) (entity.ShortLink, error) {
 	hasMapping, err := u.userShortLinkRepo.HasMapping(user, oldAlias)
 	if err != nil {
 		return entity.ShortLink{}, err
 	}
 	if !hasMapping {
-		return entity.ShortLink{}, ErrShortLinkNotFound
+		return entity.ShortLink{}, ErrShortLinkNotFound("short link not found")
```

Comment:
We need to include the alias so that we can debug when something goes wrong. `ErrShortLinkNotFound` already communicated the error type.